src/Domain/Fixture/FixtureCreator.php

```diff
@@ -211,6 +212,19 @@ private function createComposerProject(): void {
     $this->codebaseCreator->create($this->options);
   }
 
+  /**
+   * Remove "config.platform" parameter from fixture root composer.json.
+   */
+  private function removeComposerConfigPlatform(): void {
+    $this->output->writeln("Removing Platform Dependencies");
+    try {
+      $this->composer->removeConfig(['platform']);
+    }
+    catch (\Exception $e) {
+      $this->output->writeln("Operation failed.");
```

Comment:
Let's be a little more specific with our error message, e.g.,

```suggestion
      $this->output->writeln("Failed to remove Composer platform requirements.");
```

We should update the message on line 219 accordingly, e.g., "Removing Composer platform requirements". And it wouldn't hurt to rename the method to match.